dtfs-central-api/src/v1/controllers/tfm/facility/tfm-get-facilities.controller.js

```diff
@@ -30,7 +30,7 @@ exports.getAllFacilities = async (req, res) => {
   //           "facilityId": "Mock facility Id",
   //           "ukefFacilityId": "0030136443",
   //           "dealType": "GEF",
-  //           "facilityType": "Cash",
+  //           "type": "Cash",
   //           "value": 1000000,
   //           "currency": "GBP",
   //           "coverEndDate": "2021-08-12T00:00:00.000Z",
```

Comment:
Can the commented out code be removed?

Reply:
@Zainzzkk this code is just an example of what the query defined below will return - that aggregation